fbchat/get.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: UTF-8 -*-
+
+from __future__ import unicode_literals
+from .base import Base
+
+
+class Get(Base):
+    """Enables retrieving information about threads and messages"""
+
+    def get_threads(self, limit=None):
+        """Retrieve the threads that the client is currently chatting with
+
+        If ``limit`` is ``None``, the result will be a generator, otherwise the
+        result will be a list with maximum length of ``limit``
```

Comment:
I'm not sure if this is a good idea. Maybe we should just always return a list, or maybe we should remove the ``limit`` parameter and just always return an iterable, where people could shorten it as needed with [`itertools.islice`](https://docs.python.org/library/itertools.html#itertools.islice)